samples/net/sockets/civetweb/src/main.c

```diff
@@ -10,7 +10,14 @@
 
 #include "civetweb.h"
 
-#define CIVETWEB_MAIN_THREAD_STACK_SIZE 4096
+#define XSTR(x) STR(x)
+#define STR(x) #x
+
+#define HTTP_PORT	8080
+#define HTTPS_PORT	4443
+
+#define CIVETWEB_MAIN_THREAD_STACK_SIZE		CONFIG_MAIN_STACK_SIZE
+#define MAX_REQUEST_SIZE_BYTES			CONFIG_NET_TX_STACK_SIZE
```

Comment:
Hmm, how the TX stack size relates to max request size?

Reply:
My though was: requested size should not be bigger than whole TX stack size. And under request I mean - the request from the client ( not the board itself ) - therefore TX. Otherwise it should be RX. Please correct me, if I have some mistake here. You're welcome for any helpful info.

Reply:
The network packet memory is not allocated from stack so there is no relation here. The max req size default value could be set by `NET_IPV6_MTU` which is 1280 bytes, but that could/should be able to be overriden in application Kconfig file.

Reply:
Unfortunatelly, I can't put `#define MAX_REQUEST_SIZE_BYTES  CONFIG_NET_TX_STACK_SIZE` since I am doing `STRINGIFY(MAX_REQUEST_SIZE_BYTES)` later. This leads to wrong value in civetweb string buffer. Thus I just will put 1024 value for MAX_REQUEST_SIZE_BYTES as lowest border. This lowest border value follows the idea of reducing RAM usage either. Moreover I will put the comment with the notice, from where the value comes from. Please see next push.